### ISSUE TITLE
[SDK] Paragraph Height

### ISSUE THREAD

**Issue by lwnoble** [finos/a11y-theme-builder#556]
## Problem/Concern

The min wcag para graph height is 2x the text size  - so we should have a minimum value passed from the sdk of 2

I also think for the user instead of fixed pixel heights of 24px, 36px, etc. and instead show
2x 2.5x 3x 3.5 

We only need a few options there

**Comment by smithbk** [comment]
Removing "1" as an option for the paragraphPadding property on the spacing molecule
